recipe/meta.yaml

```diff
@@ -651,6 +651,46 @@ outputs:
         - LICENSE.txt
       summary: C++ libraries for Apache Parquet
 
+  - name: parquet-utils
+    script: install-libarrow.sh  # [unix]
+    script: install-libarrow.bat  # [win]
+    version: {{ version }}
+    build:
+      string: h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}_{{ build_ext }}
+    requirements:
+      build:
+        - cmake <4
+        - ninja
+        # for strong run-exports
+        - {{ stdlib("c") }}
+        - {{ compiler("c") }}
+        - {{ compiler("cxx") }}
+      host:
+        - {{ pin_subpackage("libparquet", exact=True) }}
+      run:
+        - {{ pin_subpackage("libparquet", exact=True) }}
+
+    test:
+      commands:
+        # bin
+        - test -f $PREFIX/bin/parquet-dump-arrow-statistics                    # [unix]
+        - test -f $PREFIX/bin/parquet-dump-footer                              # [unix]
+        - test -f $PREFIX/bin/parquet-dump-schema                              # [unix]
+        - test -f $PREFIX/bin/parquet-reader                                   # [unix]
+        - test -f $PREFIX/bin/parquet-scan                                     # [unix]
+        - if not exist %LIBRARY_BIN%\parquet-dump-arrow-statistics.exe exit 1  # [win]
+        - if not exist %LIBRARY_BIN%\parquet-dump-footer.exe exit 1            # [win]
+        - if not exist %LIBRARY_BIN%\parquet-dump-schema.exe exit 1            # [win]
+        - if not exist %LIBRARY_BIN%\parquet-reader.exe exit 1                 # [win]
+        - if not exist %LIBRARY_BIN%\parquet-scan.exe exit 1                   # [win]
```

Comment:
Ideally we would test that they run successfully on a sample Parquet file (that would help weed out any linkage issue), but I don't know the conda-forge common practice about this. @h-vetinari

Reply:
I'm personally always in favour of testing functionality! It would be nice to download the parquet file on the fly though, otherwise it ends up getting shipped with every install (conda packages the stuff that goes into the `test:` section as well).